alex-w/server/config/routes.js

```diff
@@ -0,0 +1,11 @@
+console.log("routes");
+var friends = require('../controllers/friends.js');
+
+module.exports = function(app){
```

Comment:
The reason whe keep calling these 'semi restful' routes, is because we're not yet using the new HTTP keywords PUT, PATCH, and DELETE. You're welcome to use PUT, PATCH, and DELETE if you want to, either way is fine!